src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -60,10 +60,12 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
             options = ''
 
         parser = argparse.ArgumentParser()
-        parser.add_argument('--cerncastor', action='store_true')
-        parser.add_argument('--old', action='store_true')
+        parser.add_argument('--wma-cerncastor', action='store_true')
+        parser.add_argument('--wma-disablewriterecovery', action='store_true')
         args, unknown = parser.parse_known_args(options.split())
 
+        # strip out WMAgent specific options
+        unknown = set(unknown) - set(['--wma-cerncastor', '--wma-disablewriterecovery'])
```

Comment:
Future not yet know parameters we might want to add later. We want to be able to just add them to site-local-config without having to worry that older code will pick them up.